datasketch/lsh.py

```diff
@@ -282,6 +301,36 @@ def _insert(
         for H, hashtable in zip(Hs, self.hashtables):
             hashtable.insert(H, key, buffer=buffer)
 
+    def __eq__(self, other:MinHashLSH) -> bool:
```

Comment:
Would be better to call it a different name. `__eq__` implies complete equivalence. This is not the case as it only checks for initialization parameters.

Reply:
Alright, will change it to something else.